spring-cloud-azure-eventhub-stream-binder/src/main/java/com/microsoft/azure/eventhub/stream/binder/properties/EventHubConsumerProperties.java

```diff
@@ -19,4 +19,5 @@
 public class EventHubConsumerProperties {
     private StartPosition startPosition = StartPosition.LATEST;
     private CheckpointMode checkpointMode = CheckpointMode.BATCH;
+    private int checkpointCount = 10;
```

Comment:
As it is magic number, could you please add some comments about why `10` ?

Reply:
Yes, will add comment for this.